main.go

```diff
@@ -183,43 +182,37 @@ func router(hh httpHandlers) http.Handler {
 type httpConfigurations struct {
 	baseURLSlice []string
 	client       http.Client
+	ticker *time.Ticker
 }
 
 func (httpConf httpConfigurations) readMessage(msg queueConsumer.Message) {
-	<-ticker.C
+	<-httpConf.ticker.C
+	ingestionType, uuid := extractMessageTypeAndId(msg.Headers)
+	err := sendToWriter(ingestionType, strings.NewReader(msg.Body), uuid, httpConf.baseURLSlice)
+
+	if err != nil {
+		log.Errorf("%v", err)
+	}
+}
+
+func extractMessageTypeAndId(headers map[string]string) (string, string) {
```

Comment:
Can this be done with a simple map lookup? Would get rid of the look and simplify things.